internal/pkg/service/buffer/event/sender.go

```diff
@@ -164,10 +164,10 @@ func (s *Sender) sendEvent(ctx context.Context, api *keboola.API, start time.Tim
 		event.Results["statistics"] = map[string]any{
 			"lastRecordAt": params.Stats.LastRecordAt.String(),
 			"recordsCount": params.Stats.RecordsCount,
-			"recordsSize":  params.Stats.RecordsSize,
-			"bodySize":     params.Stats.BodySize,
-			"fileSize":     params.Stats.FileSize,
-			"fileGZipSize": params.Stats.FileGZipSize,
+			"recordsSize":  params.Stats.RecordsSize.Bytes(),
+			"bodySize":     params.Stats.BodySize.Bytes(),
+			"fileSize":     params.Stats.FileSize.Bytes(),
+			"fileGZipSize": params.Stats.FileGZipSize.Bytes(),
```

Comment:
Pouzity `datasize` type je sice zalozeny na `uint64` ale serializuje sa do JSON ako `string`.

V [JSON scheme](https://github.com/keboola/event-schema/blob/130a890c67494dd9d22a769e5c35d9b86713010e/schema/ext.keboola.keboola-buffer..json#L33-L37) mame `int`.

Reply:
Samo o sebe to nepomohlo, kedze sme to posielali ako `form-data` v `go-client`.

Ale ked sa to preplo na `JSON body`, tak to uz ide, vid:
https://keboola.slack.com/archives/C02JAQXLY92/p1677853344346269